### PR TITLE
feat(LOC-221): add Avatar component

### PR DESCRIPTION
**Pull request by flycrum**
**Audience:** Engineers | Third-party Developers

**Summary:** Added an `Avatar` component with image and initials visuals, team and user types, background colors, and sizes. To support this feature, a `ClippedContent` component has been created and the `ImageCircle` was modified to use `ClippedContent`.

**Screenshots:** 
image
image
image

### CODE REVIEW

File: src/components/ImageCircle/ImageCircle.sass [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import '../../styles/_partials/index'
 
-.ImageCircleContainer
+body .ImageCircleContainer
```
Comment (flycrum) [inline src/components/ImageCircle/ImageCircle.sass]: This is an unfortunate workaround having to deal with `styles` added by a developer. There's certainly a better solution, but I didn't want it to slow down progress.

File: src/components/ImageCircle/ImageCircle.tsx [hunk 2]
```diff
@@ -22,17 +26,20 @@ export default class ImageCircle extends React.Component<IProps> {
 	};
 
 	render () {
-		const ContainerTag: any = this.props.tag;
-
 		return (
-			<ContainerTag
+			<ClippedContent
```
Comment (flycrum) [inline src/components/ImageCircle/ImageCircle.tsx]: Now using `ClippedContent` internally so its consistent sizing and shape with Avatar initials.

File: src/components/VerticalNav/VerticalNav.tsx [hunk 3]
```diff
@@ -145,6 +146,7 @@ interface IWorkspaceSwitcherProps extends IReactComponentProps {
 	onClickAddTeam: Handler;
 	onClickLogout: Handler;
 	onClickWorkspace: Handler;
+	onClickWorkspaceNav: Handler;
```
Comment (flycrum) [inline src/components/VerticalNav/VerticalNav.tsx]: This eventual is specifically for when the `VerticalNav` icon is clicked.

File: src/components/ImageCircle/ImageCircle.sass (hunk 1 above)
Comment (claygriffiths) [reply src/components/ImageCircle/ImageCircle.sass]: Yeah, I'd like to see this eliminated if possible.

Comment (flycrum) [reply src/components/ImageCircle/ImageCircle.sass]: Good call. Done. Added some props to remove the need for this.

**Pull request merged**